apps/console/src/features/applications/components/application-list.tsx

```diff
@@ -374,8 +375,11 @@ export const ApplicationList: FunctionComponent<ApplicationListPropsInterface> =
                     const hasScopes: boolean = !hasRequiredScopes(featureConfig?.applications,
                         featureConfig?.applications?.scopes?.delete, allowedScopes);
 
```

Comment:
Let's remove this empty line. Can only keep line before the return statement (380)

Reply:
Resolved comment with the PR : 
https://github.com/wso2/identity-apps/pull/3253